pinot-plugins/pom.xml

```diff
@@ -146,6 +146,10 @@
                       <pattern>org.apache.commons</pattern>
                       <shadedPattern>shaded.org.apache.commons</shadedPattern>
                     </relocation>
+                    <relocation>
+                      <pattern>org.apache.avro</pattern>
+                      <shadedPattern>shaded.org.apache.avro</shadedPattern>
```

Comment:
should we consider using `${shade.prefix}` instead of a fixed `shaded` prefix?

Reply:
You are right! I brainlessly copied from the previous shade block! Updated both.